### PR TITLE
Refactor analytics_api to use edx_service

### PR DESCRIPTION
**Pull request by maxrothman**

### CODE REVIEW

**Comment by feanil** [comment]
Since you renamed the role, you'll need to update all playbooks that call this role as well.

**Comment by maxrothman** [comment]
``````
configuration/playbooks $ grep -R analytics-api *
roles/analytics_api/defaults/main.yml:18:ANALYTICS_API_NEWRELIC_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-analytics-api"
roles/analytics_api/defaults/main.yml:26:    NAME: 'analytics-api'
roles/edxlocal/tasks/main.yml:106:- name: create read-only reports user for the analytics-api
roles/insights/defaults/main.yml:108:INSIGHTS_NEWRELIC_APPNAME: "{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-analytics-api"
roles/mariadb/defaults/main.yml:37:  - "{{ ANALYTICS_API_CONFIG['DATABASES']['default']['NAME']|default('analytics-api') }}"
roles/mariadb/defaults/main.yml:99:    priv: "{{ ANALYTICS_API_CONFIG['DATABASES']['default']['NAME']|default('analytics-api') }}.*:ALL"```
``````

**Comment by fredsmith** [comment]
:+1:

**Comment by maxrothman** [comment]
@feanil does that address your concern?

File: playbooks/edx-east/analyticsapi.yml [hunk 1]
```diff
@@ -9,9 +9,9 @@
   roles:
     - role: nginx
       nginx_sites:
-        - analytics-api
+        - analytics_api
```
Comment (feanil) [inline playbooks/edx-east/analyticsapi.yml:12]: You changed the name of the NGINX site, but it looks like you didn't update the name of the relevant template.  Take a look (here)[https://github.com/edx/configuration/blob/master/playbooks/roles/nginx/tasks/main.yml#L67-L73] to see how this gets used.

**Comment by feanil** [comment]
:+1:

**Comment by e0d** [comment]
:+1:  Seems good, think we may need to change some tags in order to get AMIs to build correctly.

**Comment by maxrothman** [comment]
@e0d can you elaborate?

**Pull request merged**